ote/src/cljs/ote/views/admin/monitor.cljs

```diff
@@ -71,13 +71,19 @@
     (let [translate-typekw (fn [type]
                               (tr [:enums :ote.db.transport-service/sub-type (keyword type)]))
            companies-by-month-data {:labels (mapv :month (:monthly-companies monitor-data))
-                                    :datasets [{:label "Liikkumispalveluiden tarjoajien lukumäärä"
-                                                :data (mapv :sum (:monthly-companies monitor-data))
-                                                :backgroundColor "rgb(0, 170, 187)"}]}
+                                    :datasets [{:label "Palveluntuottajat"
+                                                :data (mapv :sum-providing (:monthly-companies monitor-data))
+                                                :backgroundColor "rgb(0,136,160)"}
+                                               {:label "Palveluiden tuottamiseen osallistuvat yritykset"
+                                                :data (mapv :sum-participating (:monthly-companies monitor-data))
+                                                :backgroundColor "rgb(102,204,214)"}]}
            companies-by-tertile-data {:labels (mapv :tertile (:tertile-companies monitor-data))
-                                      :datasets [{:label "Liikkumispalveluiden tarjoajien lukumäärä"
-                                                  :data (mapv :sum (:tertile-companies monitor-data))
-                                                  :backgroundColor "rgb(0, 170, 187)"}]}
+                                      :datasets [{:label "Palveluntuottajat"
+                                                  :data (mapv :sum-providing (:tertile-companies monitor-data))
+                                                  :backgroundColor "rgb(0,136,160)"}
+                                                 {:label "Palveluiden tuottamiseen osallistuvat yritykset"
+                                                  :data (mapv :sum-participating (:tertile-companies monitor-data))
+                                                  :backgroundColor "rgb(102,204,214)"}]}
```

Comment:
Should these be the other way around? So that "Palveluntuottajat" label is at the bottom like it is in the chart?